web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -157,6 +157,11 @@ class TabbedEditor extends Component {
 
     // TODO consider also using command+t (which currently conflicts with a CM command)
     openFileSearch: 'command+p',
+    nextTab: 'command+]',
```

Comment:
Conflicts with indentation hotkeys.

Chrome, Finder, and most other applications use `command+shift+]` instead, while `command+]` means "back". Atom/Sublime also use `command+shift+]` with the default keybindings, so lets definitely have `command+shift+]`. I'm also fine with having a `command+]` that only works in VIM mode, if Atom/Sublime do that.

Does `command+shift+]` conflict with VIM mode?